src/index.html

```diff
@@ -13,7 +13,7 @@
     <meta http-equiv="Cache-Control" content="public" />
     <meta http-equiv="Pragma" content="no-cache" />
     <meta http-equiv="Expires" content="0" />
-    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=5, viewport-fit=cover">
+    <meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no, maximum-scale=5, viewport-fit=cover">
```

Comment:
Hi @SailingSteve, we needed user-scalable to be allowed so people can increase the font size. Was setting it to user-scalable=no critical for a problem you found?

Reply:
I removed that an hour ago on my local, it was in the wrong index.html, so next PR will back that out.